commit 055d0b77e1e2823daef60a4c2b201531dc0c7248
Author: Panu Matilainen <pmatilai@redhat.com>
Date:   Mon Mar 1 10:58:17 2010 +0200

    Don't run brp-strip-comment-note when find-debuginfo.sh is run (#568924)
    - brp-strip-comment-note is not only unnecessary here but is also
      now messing up things by resetting EI_OSABI to zero (#568921)
    - patch from  Roland McGrath

diff --git a/macros b/macros
index 7721fe0..1ac46d7 100644
--- a/macros
+++ b/macros
@@ -92,9 +92,11 @@
 
 %__os_install_post    \
     /usr/lib/rpm/redhat/brp-compress \
-    %{!?__debug_package:/usr/lib/rpm/redhat/brp-strip %{__strip}} \
-    /usr/lib/rpm/redhat/brp-strip-static-archive %{__strip} \
+    %{!?__debug_package:\
+    /usr/lib/rpm/redhat/brp-strip %{__strip} \
     /usr/lib/rpm/redhat/brp-strip-comment-note %{__strip} %{__objdump} \
+    } \
+    /usr/lib/rpm/redhat/brp-strip-static-archive %{__strip} \
     /usr/lib/rpm/brp-python-bytecompile %{__python} %{?_python_bytecompile_errors_terminate_build} \
     /usr/lib/rpm/redhat/brp-python-hardlink \
     %{!?__jar_repack:/usr/lib/rpm/redhat/brp-java-repack-jars} \
